Source/Scene/Cesium3DTile.js

```diff
@@ -120,6 +120,23 @@ define([
         this.children = [];
 
         /**
+         * Descendant tiles that need to be visible before this tile can refine. For example, if
+         * a child is empty (such as for accelerating culling), its descendants with content must
+         * be loaded first. If a tiles's children all have content, this is left undefined.
+         *
+         * @type {Array}
+         * @readonly
+         */
+        this.descendantsWithContent = undefined;
+
+        /**
+         * Marks if the tile is selected this frame.
+         *
+         * @type {Boolean}
+         */
+        this.selected = false;
```

Comment:
Move this down to the section
```
// Members that are updated every frame for rendering optimizations:
```
and perhaps update that comment since the dynamic per-frame members are now for more than optimization.